app/templates/components/add-cron-job.hbs

```diff
@@ -1,32 +1,64 @@
-<form {{action (perform save) on="submit"}}>
-  <div class="form-elem select">
-    <span class="label">Branch</span>
-    {{#x-select value=selectedBranch required="true" on-change=(action (mut selectedBranch)) as |xs|}}
-      {{#each branches as |branch|}}
-        {{#xs.option value=branch}}{{branch.name}}{{/xs.option}}
-      {{/each}}
-    {{/x-select}}
+{{#travis-form
+  onSubmit=(perform save)
+  as |form|
+}}
+  <div class='form-elem select'>
+    {{#form.field
+      label='Branch'
+      value=selectedBranch
+      required=true
+      class='branch-selector travis-form__field--without-badge'
+      as |field|
+    }}
+      {{#field.select
+        searchField='name'
+        placeholder='Type branch name'
```

Comment:
I would like suggest a bit different labels: https://github.com/travis-ci/travis-web/blob/2adf33c31687303b6770724f7d1e65a86e3c59ba/app/templates/components/add-cron-job.hbs#L7-L21

Reply:
@weban could you please comment on this?